src/js/components/search/newResultsView/SearchSectionWrapper.jsx

```diff
@@ -103,15 +103,15 @@ const SearchSectionWrapper = ({
                 closedIcon="chevron-down"
                 openIcon="chevron-up"
                 title="Data sources and methodology" >
-                {openAccordion && (
+                {openAccordion ? (
```

Comment:
I had noticed a console error related to children being boolean values instead of an html/react elements. This was my attempt to clear it. Although I am now having trouble replicating the original error consistently.